cellprofiler/preferences.py

```diff
@@ -257,8 +257,10 @@ def config_exists(key):
         return True
     if not get_config().Exists(key):
         return False
-    if get_config().GetEntryType(key) == 1:
-        return get_config().Read(key) is not None
+    # FIXME: Issue reported at https://github.com/wxWidgets/Phoenix/issues/1292; quick hack for it to work for now
```

Comment:
I think we don't want to evade checking entirely on Windows, just in case we "forget"/deprioritize coming back to this; can you add an "else" that checks the text written to the config and handles it appropriately? For almost any other preference I wouldn't bother, but it's the "Can we track your usage" preference, so because of GDPR etc we don't want to ignore the user's stated intent.

Reply:
Sure; just a head's up, the fix I'm thinking isn't very pretty. The pseudo-code is something on the lines of

> else:
> => if key == some_bool_key:
> ===> return get_config().ReadBool(key) is not None
> => elif key == some_int_key>:
> ===> return get_config().ReadInt(key) is not None
> => etc...

Would this be alright for the time being?

Reply:
@bethac07 Just a head's up that with the new changes, on Windows, it will use the respective `.Read(key)` to check if the key exists.